<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-20</y>
      <width>981</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>third_page_2.webp</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>811</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
    background-color: rgba(255, 255, 255, 0.9); /* White background with 80% opacity */
    border-radius: 50px; /* Rounded corners */
    padding: 20px; /* Optional: Add some padding inside the box */
    margin: 20px; /* Optional: Add margin around the box */
    color: #000000; /* Text color inside the box */
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>471</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">FONT-SIZE:30PX;</string>
    </property>
    <property name="text">
     <string>SOLUTION</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>470</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #50a9b8, stop:1 #62ba74); /* Blue to black as main theme */
    border-radius: 25px; /* Optional: for rounded corners */
    border: none; /* Optional: remove border */
    color: white; /* Text color */
    padding: 10px 20px; /* Optional: adjust padding */
    font-size: 16px; /* Optional: adjust font size */
    transition: background 10s; /* Super slow transition */
}

QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #62ba74, stop:1 #50a9b8); /* Black to blue on hover */
}
</string>
    </property>
    <property name="text">
     <string>SOLVE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
                background-color: transparent;
                border: none; /* Remove default border */
                padding: 0;
                icon-size: 100px; /* Adjust icon size if needed */
            }
            QPushButton:hover {
                background-color: rgba(0, 0, 0, 0.1); /* Slightly darkened background */
                border: none;
            }
            QPushButton::icon {
                padding: 0;
                border-image: url(path_to_your_image.png); /* Set image */
            }
            QPushButton:hover::icon {
                border-image: url(path_to_your_hover_image.png); /* Blurred or darkened image on hover */
            }
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Downloads/DALL_E_2024-09-04_17.45.07_-_Create_several_spherical_buttons_with_a_back_arrow_symbol_in_a_style_that_matches_the_colorful_and_creative_physics-themed_image_filled_with_equations__1_-modified-removeb.png</normaloff>../../Downloads/DALL_E_2024-09-04_17.45.07_-_Create_several_spherical_buttons_with_a_back_arrow_symbol_in_a_style_that_matches_the_colorful_and_creative_physics-themed_image_filled_with_equations__1_-modified-removeb.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">FONT-SIZE:25PX;</string>
    </property>
    <property name="text">
     <string>Eigen Value ⇒</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">FONT-SIZE:25PX;</string>
    </property>
    <property name="text">
     <string>Eigen Vector ⇒</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>230</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">FONT-SIZE:20PX;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>290</y>
      <width>501</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">FONT-SIZE:16PX;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
